.travis.yml

```diff
@@ -17,26 +17,13 @@ language: python
 matrix:
   include:
 
-    # Use trusty dist for python 2.6, since the built-in xenial distro (default)
-    # does not support Python 2.6
-    # See https://docs.travis-ci.com/user/reference/xenial/#python-support
-
-    # Python 2.6, Server #1
```

Comment:
Python 2.6 was deprecated on May 1rst and the build is failing, so let's get rid of it.